app/upgrades/vNEXT.go

```diff
@@ -31,13 +74,112 @@ var Upgrade_NEXT = Upgrade{
 		keepers *keepers.Keepers,
 		configurator module.Configurator,
 	) upgradetypes.UpgradeHandler {
-		// Add new parameters by:
-		// 1. Inspecting the diff between vPREV..vNEXT
-		// 2. Manually inspect changes in ignite's config.yml
-		// 3. Update the upgrade handler here accordingly
-		// Ref: https://github.com/pokt-network/poktroll/compare/vPREV..vNEXT
+		setIBCParams := func(ctx context.Context) (err error) {
+			sdkCtx := cosmostypes.UnwrapSDKContext(ctx)
+
+			estimatedBlockDuration, isFound := pocket.EstimatedBlockDurationByChainId[sdkCtx.ChainID()]
+			if !isFound {
+				return fmt.Errorf("chain ID %s not found in EstimatedBlockDurationByChainId", sdkCtx.ChainID())
+			}
```

Comment:
Please add some logging for the values we're extracting here.

Will need to be really careful during the upgrade.